### ISSUE TITLE
New Vercel Deployment Re: Nextjs Config

### ISSUE THREAD

**Issue by PerfectProductions** [blitz-js/blitz#4200]
### What is the problem?

Hi there bugzappers!

So today Ive attempted to deploy against vercel and it appears that the "Target:" property has recently been removed from Nextjs, leaving us all with a little issue regarding vercel deployments.

Please see the below screenshot for a more accuratly logged issue!

https://tinyurl.com/258xf9rq

### Paste all your error logs here:

```
PASTE_HERE (leave the ``` marks)
```

### Paste all relevant code snippets here:

```
PASTE_HERE (leave the ``` marks)
```

### What are detailed steps to reproduce this?

1. Simply attempt to deploy your Blitzjs Application to Vercel.

### Run `blitz -v` and paste the output here:

```
PASTE_HERE (leave the ``` marks)
```

### Please include below any other applicable logs and screenshots that show your problem:

I have no idea what to do about this so I've left this to the crew to provide updates and direction.

Thanks Crew!

**Comment by siddhsuresh** [comment]
@PerfectProductions I think you choose the legacy blitz preset, since blitz 2.0 is now a toolkit, you can use the next.js preset. Let me know if this works for you

**Comment by PerfectProductions** [comment]
Thanks I ... didn't get to choose a preset when deploying to vercel, where may I choose the nextjs preset?

Blitz version: 2.0.0-beta.31 (global)
Blitz version: 2.0.0-beta.31 (local)
Linux 6.1 | linux-arm64 | Node: v20.2.0


 Package manager: npm

  System:
    OS: Linux 6.1 Gentoo Linux
    CPU: (4) arm64 Neoverse-N1
    Memory: 14.13 GB / 23.44 GB
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 20.2.0 - ~/.nvm/versions/node/v20.2.0/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v20.2.0/bin/yarn
    npm: 9.7.1 - ~/.nvm/versions/node/v20.2.0/bin/npm
  npmPackages:
   ***@***.***/auth: 2.0.0-beta.31 => 2.0.0-beta.31
   ***@***.***/next: 2.0.0-beta.31 => 2.0.0-beta.31
   ***@***.***/rpc: 2.0.0-beta.31 => 2.0.0-beta.31
   ***@***.***/client: 4.6.1 => 4.6.1
    blitz: 2.0.0-beta.31 => 2.0.0-beta.31
    next: 13.4.5 => 13.4.5
    prisma: 4.6.1 => 4.6.1
    react: 18.2.0 => 18.2.0
    react-dom: 18.2.0 => 18.2.0
    typescript: 5.1.6 => 5.1.6

Sent with Spark

**Comment by PerfectProductions** [comment]
That is so true! Thank you for the redirect! All done! 

For what its worth! I thought this was directly related to my build however I was able to overcome dynamic imports and successfully produce a lovely blitz motif. 

Once I had that all sorted out I was able to deploy to vercel using your advice to select the nextjs platform rather than the legacy template. 

Everything went off without a hitch! Finally, my first Blitzjs Production Build, loading time down from 11.8 seconds using CKEditor to 1.6 under production build. 

Blitz is a seriously decent framework. Well done Brandon and Crew! 

On a Sidenote:
Will close this and try discord for future issues of this nature!

Thank you @siddhsuresh - Again - Thankyou! :)
